molr-mole-core/src/main/java/io/molr/mole/core/support/OngoingMissionRun.java

```diff
@@ -1,32 +1,37 @@
 package io.molr.mole.core.support;
 
-import io.molr.commons.domain.MissionHandle;
-import io.molr.commons.domain.MissionState;
-import io.molr.commons.domain.Result;
-import io.molr.commons.domain.RunState;
+
+import io.molr.commons.domain.*;
 import io.molr.mole.core.api.Mole;
+import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 import java.time.Duration;
+import java.util.function.Function;
 
 import static java.util.Objects.requireNonNull;
 
+/**
+ *
+ */
 public class OngoingMissionRun {
 
-    private final Mole mole;
-    private final Mono<MissionHandle> handle;
+    protected final Mole mole;
+    protected final Mono<MissionHandle> handle;
```

Comment:
Why are these fields protected?

Reply:
OngoingReturningMissionRun is inheriting this class. So to let it access the fields, I've made them protected.